mm2src/coins/lp_coins.rs

```diff
@@ -499,6 +523,17 @@ pub trait SwapOps {
         swap_unique_data: &[u8],
     ) -> TransactionFut;
 
+    fn send_watcher_spends_maker_payment(&self, _input: WatcherSpendsMakerPaymentInput) -> TransactionFut;
+
+    fn sign_maker_payment(
```

Comment:
`sign_maker_payment` name is confusing and not reflecting the actual purpose of this method. As of now, it signs the *taker spending maker payment* tx, not maker payment itself. Also, as per https://github.com/KomodoPlatform/atomicDEX-API/issues/1431#issuecomment-1204883845, I assume it can return the entire spending transaction (without secret appended) and send it to watcher(s). They can then append the extracted secret to HTLC input's script_sig. It will hopefully allow simplifying `send_watcher_spends_maker_payment` or maybe removing it at all in favor of (pseudocode):
```
let final_spend_tx = maker_coin.append_secret(preimage, secret);
maker_coin.send_raw_tx(final_spend_tx);
```